cloudfoundry-incubator/stratos
# Restructure go backend

This PR restructures the go backend to make it easier to develop and switches to using dep for dependency management. gulp is no longer used to build the backend.

DO NOT MERGE THIS PR - I will create a PR with the source file location changes - this PR contains a script to move the source files around as well as the changes to the build files for this new structure. The source file moves are omitted from this PR to make it easier to review the other changes.

To test:

- Check out this branch
- From the top-level folder, run this script: ./build/tools/golang-movesrc.sh

If you wish to check a local compile, ensure the Stratos repo is cloned into the appropriate location within your GOPATH, e.g:

$GOPATH/src/github.com/cloudfoundry-incubator/stratos
